marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/entities/search/getEntityInstance.sjs

```diff
@@ -0,0 +1,117 @@
+/**
+ Copyright (c) 2020 MarkLogic Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+'use strict';
+
+const entitySearchLib = require("/data-hub/5/entities/entity-search-lib.sjs");
+const test = require("/test/test-helper.xqy");
+
+const shippingResults = [
+  {
+    "Address": {
+      "street": "Whitwell Place",
+      "city": "Ellerslie",
+      "state": "Georgia",
+      "zip": {
+        "Zip": {
+          "fiveDigit": "52239",
+          "plusFour": "1718"
+        }
+      }
+    }
+  },
+  {
+    "Address": {
+      "street": "Skyway road",
+      "city": "San carlos",
+      "state": "California",
+      "zip": {
+        "Zip": {
+          "fiveDigit": "94070",
+          "plusFour": "1234"
+        }
+      }
+    }
+  }
+];
+
+const billingResults = {
+  "Address": {
+    "street": "Anna Court",
+    "city": "Stewart",
+    "state": "Kansas",
+    "zip": {
+      "Zip": {
+        "fiveDigit": "62601",
+        "plusFour": "6783"
+      }
+    }
+  }
+};
+
+function getInstanceWithoutESInfo() {
+  let docUri = "/content/sally.json";
+  const instanceArray = entitySearchLib.getEntityInstance(docUri);
+  const results = instanceArray[0]["Customer"].toObject();
+  return [
+    test.assertEqual(101, results["customerId"]),
+    test.assertEqual("Sally Hardin", results["name"]),
+    test.assertEqual(shippingResults, results["shipping"]),
+    test.assertEqual(["Sal", "din", "shh"], results["nicknames"]),
+    test.assertEqual(JSON.stringify(billingResults), JSON.stringify(results["billing"]))
+  ];
+}
+
+function getInstanceWithESInfo() {
+  let docUri = "/content/jane.json";
+  const instanceArray = entitySearchLib.getEntityInstance(docUri);
```

Comment:
These looks like good tests, but hoping that getEntityInstance can be adjusted above to either return a single object or nothing, not an array.